src/04-modality-specific-files/09-positron-emission-tomography.md

```diff
@@ -0,0 +1,446 @@
+
+## Terminology and conventions
+
+PET-BIDS is fully consistent with the BIDS specification as a whole.
+However, BIDS was initially developed in the context of MRI,
+so some terminology may be unfamiliar to researchers from each field.
+This section adds clarifications to
+[Common Principles - Definitions](../02-common-principles.md#definitions)
+for the PET context, and introduces the term "time zero" which is currently
+specific to PET.
+
+1.  **Session** - In most cases, a new session with respect to PET corresponds
+    to a visit to the scanning site, and starts with a new injection.
+    In situations where different data types are obtained over several visits
+    (for example, FDG PET on one day followed by amyloid PET a couple days after)
+    these scans may be grouped into the same session.
+    In other datasets, a subject leaving the scanner and returning under the same
+    injection may be considered separate sessions.
+
+1.  **Run** - In PET, subjects may have to leave the scanner to use the bathroom.
+    While leaving the scanner would interrupt an MR acquisition, in PET this
+    disruption is more appropriately considered missing data during a run.
+
+1.  **Time zero** - A reference point in time, to which all timestamps pertaining
+    to a recording are relative.
+    Time zero will most commonly be the time of injection of a radioisotope, or
+    the time at which the first scan of an acquisition is begun.
+    If a pharmacological within-scan challenge is performed,
+    another time zero may be more convenient.
+
+An overview of a common PET experiment (with blood data) can be seen in Figure 1,
+defined on a single time scale relative to a predefined “time zero”.
+
+![Figure 1](images/PET_scan_overview.svg "Overview of PET recording data")
+
+Figure 1: Overview of a common PET experiment, including blood measurements,
+and defined on a common time scale.
+Note, “time zero” can either be defined as time of injection or scan start,
+and all the PET and blood data should be decay-corrected to this time point.
+In this example, tracer injection coincides with scan start.
+
+## PET recording data
+
+Template:
+
+```Text
+sub-<label>/[ses-<label>/]
+    pet/
+        sub-<label>[_ses-<label>][_task-<label>][_acq-<label>][_rec-<label>][_run-<index>]_pet.nii[.gz]
+        sub-<label>[_ses-<label>][_task-<label>][_acq-<label>][_rec-<label>][_run-<index>]_pet.json
+```
+
+PET data belong in the `pet` directory.
+PET imaging data SHOULD be stored in 4D (or 3D, if only one volume was acquired)
+NIfTI files with the `_pet` suffix.
+Volumes MUST be stored in chronological order (the order they were acquired in).
+
+The OPTIONAL [`task-<label>`](../99-appendices/09-entities.md#task) is used to
+indicate a task subjects were asked to perform in the scanner.
+Those labels MUST be consistent across subjects and sessions.
+
+The [`acq-<label>`](../99-appendices/09-entities.md#acq) entity is used to
+indicate the tracer used.
```

Comment:
Maybe `tracer` would be a better and more understandable word to use?

Reply:
That would involve adding a new entity, which I would recommend against unless you can help it.

Reply:
I agree `ce` is not a good acronym and might be confusing for PET people, so I would vote for sticking with `acq`.

Reply:
That's fair. @mnoergaard's idea of a new entity (`tracer`) may be the best solution in the long run, but I think it would be best to implement that in 2.0 rather than add a new entity to 1.6. We have a separate repository for 2.0 proposals where I can open an issue about this.

Reply:
Hi @tsalo , does this though mean that we should stick with `acq` for now? Do we agree on this?

Reply:
Hi all, I had a chat with M. Veronese about the `acq` vs `tracer` label and we both agree that, in our view, there is no difference in sticking with the one or adopting the other as long as it is unequivocal which tracer was used if there are multiple tracers.
If then the first version of the standard sticks with one and a future will use the other the adaptation of the older datasets would be fairly easy with any file system operation.
Hope this is somehow useful as we wanted to state our opinion "for the record".

Reply:
Thank you!

Reply:
Based on the discussion in bids-standard/bids-2-devel#46, I support either approach. Since it sounds like the team can converged on `acq`, I'll resolve this discussion thread. Thanks for your patience!